django_slack/__init__.py

```diff
@@ -288,6 +288,23 @@
 
 This allows you to, for example, use the Requests library to whilst sending
 your message asynchronously via Celery.
+
+Testing
+-------
+
+You can set ``SLACK_BACKEND = django_slack.backends.TestBackend`` when testing
+to store messages to inspect them later::
+
+    from django_slack.utils import get_backend as get_slack_backend
```

Comment:
Why ``as`` here? :)  Seems to be a distraction in the docs, although I can see why one might do this in practice.